### DISCUSSION TITLE
Umrechnungsfaktor bei Integer

### DISCUSSION THREAD

**Discussion by Dachmelker** [camueller/SmartApplianceEnabler#608] [Installation und Betrieb]
Hallo Axel, Hallo zusammen

 der angezeigte Wert von **210** sind real **2,1**
wie kann ich ich das im SAE umrechnen
mein Faktor von 0.001 funktioniert leider nicht.
auch ein umstellen auf "Integer Float" schafft keine Abhilfe.

Ich weiß nicht mehr weiter 

![SAE Semp](https://github.com/camueller/SmartApplianceEnabler/assets/122626002/dbbcf91c-4426-42c4-9e2e-084f7c81a5d1)

![SAE Zähler](https://github.com/camueller/SmartApplianceEnabler/assets/122626002/cea2d275-4f59-46ec-b6ae-2100fa0f60e1)

![Sunny Portal](https://github.com/camueller/SmartApplianceEnabler/assets/122626002/55ce5bf9-3554-4ba7-be51-9444e3ccd185)

**Comment by camueller** [comment]
Zeig doch mal die Zeile im Log-File (suche nach ValueExtractor), damit man das nachvollziehen kann. Die Zeile müsste etwa so aussehen:
```
2024-05-22 19:57:09,795 DEBUG [MQTT Call: F-00000001-000000000014-00-MqttElectricityMeter-0] d.a.s.u.ValueExtractor [ValueExtractor.java:69] F-00000001-000000000014-00: value=782.91632 inputValue=782.91632 valueExtractionRegex=null extractedValue=null factorToValue=null
```

**Comment by Dachmelker** [comment]
ich hoffe du erkennst was?

![sae log](https://github.com/camueller/SmartApplianceEnabler/assets/122626002/47e11220-7682-4667-b109-dc63b8a1a365)

**Comment by Dachmelker** [comment]
Hallo Axel, 

hat sich erledigt!!

Float als Integer funktioniert jetzt.

jetzt werden 2 Watt angezeigt, 

![SAE i O](https://github.com/camueller/SmartApplianceEnabler/assets/122626002/b8936903-4dd3-416c-b322-2a81c73902e8)
